<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPointDialog</class>
 <widget class="QDialog" name="EditPointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование пункта</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>11</y>
     <width>101</width>
     <height>147</height>
    </rect>
   </property>
   <property name="title">
    <string>Hidden fields</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="editStartDate">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editOpenDate">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>336</width>
     <height>214</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Открытие</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dateEditOpen"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Начало работы</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEditStart"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Конец смены</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="timeEditEndOfShift">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>21</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>График</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxSchedule"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonApply">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonClose">
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editPointName"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Площадка</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxMarketplace">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>Wildberries</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ozon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Яндекс.Маркет</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Сотрудники</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFirstEmployee"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxSecondEmployee"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editPointName</tabstop>
  <tabstop>comboBoxFirstEmployee</tabstop>
  <tabstop>comboBoxMarketplace</tabstop>
  <tabstop>comboBoxSecondEmployee</tabstop>
  <tabstop>dateEditOpen</tabstop>
  <tabstop>dateEditStart</tabstop>
  <tabstop>timeEditEndOfShift</tabstop>
  <tabstop>comboBoxSchedule</tabstop>
  <tabstop>buttonApply</tabstop>
  <tabstop>buttonClose</tabstop>
  <tabstop>editStartDate</tabstop>
  <tabstop>editOpenDate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
